Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -57,6 +57,25 @@ public Orientation Orientation
                 new PropertyMetadata(Orientation.Horizontal, OrientationPropertyChanged));
 
         /// <summary>
+        /// Gets or sets the menu flyout placement
+        /// </summary>
+        public MenuFlyoutPlacement FlyoutPlacement
```

Comment:
I actually agree with @skendrot on this one, It just seems like something that would be more trouble than worth. Should be all handled auto.